app/rpc.js

```diff
@@ -37,6 +37,48 @@
         "thread-open": async function(threadId) {
             await F.mainView.openThreadById(threadId);
         },
+         
+        "thread-call-start": async function(callId, options) {
+            options = options || {};
+            const threadId = options.threadId || callId;
+            const thread = requireThread(threadId);
+            const callMgr, callId = F.calling.getOrCreateManager(callId, thread);
+            console.log(callMgr, callId);
+            await callMgr.start(options);
+        },
+ 
+        "thread-call-join": async function(callId) {
+            const callMgr = F.calling.getManager(callId);
+            if (!callMgr) {
+                throw new ReferenceError("Invalid callId");
+            }
+            if (!callMgr.view) {
+                throw new TypeError("Call does not have bound view");
+            }
+            await callMgr.view.join();
+        },
+ 
+        "thread-call-leave": async function(callId) {
+            const callMgr = F.calling.getManager(callId);
+            if (!callMgr) {
+                throw new ReferenceError("Invalid callId");
+            }
+            if (!callMgr.view) {
+                throw new TypeError("Call does not have bound view");
+            }
+            await callMgr.view.leave();
+        },
+ 
+        "thread-call-end": async function(callId) {
+            const callMgr = F.calling.getManager(callId);
+            if (!callMgr) {
+                throw new ReferenceError("Invalid callId");
+            }
+            if (!callMgr.view) {
+                throw new TypeError("Call does not have bound view");
+            }
+            await F.CallView.remove();
```

Comment:
This is calling the remove function of a CallView class function.

I think perhaps you meant to use `callMgr.view.remove();`

Reply:
Thanks, I was a bit sloppy with the cleanup this time! Will look closer next time